dashboard/assets/common/common.css

```diff
@@ -1,18 +1,3 @@
-#content {
```

Comment:
OK so `main.css` affects the non Django-admin front-end, and `common.css` is, as the name suggests, common to both applications. In the main dashboard app we don't have any pages (to my knowledge) where there are tables that stretch from edge to edge when the screen is small (e.g. 900px) but in admin, we do. So I moved the chunk of CSS that was applying padding globally and reapplied it to sections of admin that need it.

I see now that I also moved the min-height bits without realising. I shall move those back. 😃